smart-contract/program/src/processor/unstake.rs

```diff
@@ -42,19 +38,28 @@ pub struct Accounts<'a, T> {
     /// The owner of the stake account
     #[cons(signer)]
     pub owner: &'a T,
+
+    /// Optional bond account to be able to stake under the minimum
+    pub bond_account: Option<&'a T>,
 }
 
 impl<'a, 'b: 'a> Accounts<'a, AccountInfo<'b>> {
     pub fn parse(
         accounts: &'a [AccountInfo<'b>],
         program_id: &Pubkey,
+        has_bond_account: bool,
     ) -> Result<Self, ProgramError> {
         let accounts_iter = &mut accounts.iter();
         let accounts = Accounts {
             central_state_account: next_account_info(accounts_iter)?,
             stake_account: next_account_info(accounts_iter)?,
             stake_pool: next_account_info(accounts_iter)?,
             owner: next_account_info(accounts_iter)?,
+            bond_account: if has_bond_account {
```

Comment:
This is equivalent to just writing `bond_account: next_account_info(accounts_iter).ok()`. `has_bond_account` is not required

Reply:
Good point - fixed